lib/generators/solidus_auth_devise/install/install_generator.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module SolidusAuthDevise
+  module Generators
+    class InstallGenerator < Rails::Generators::Base
+      class_option :auto_run_migrations, type: :boolean, default: false
+
+      def add_javascripts
```

Comment:
js and css files + these install methods should be removed, right?

Reply:
@kennyadsl done! Actually, the entire generator had to be removed since we already had one at a different path.